client/Configurator.c

```diff
@@ -455,6 +455,7 @@ static void security(struct Allocator* tempAlloc, List* conf, struct Log* log, s
     }
 
     int uid = -1;
+    int64_t* group;
```

Comment:
Please initiate this to NULL so I don't have to go looking around the file to see if ```if (group)``` is going to be checking an uninitiated value...